content/blog/2021-07-03-darktable-3.6/index.md

```diff
@@ -0,0 +1,217 @@
+
+## Demosaic & Details Mask
+
+### New demosaicing algorithms
+
+A new [Ratio-Corrected Demosaicing](https://github.com/LuisSR/RCD-Demosaicing) (RCD) algorithm has been added for Bayer array sensors. This is essentially an improvement over the AMaZE method, offering almost the same amount of detail recovery but with fewer artifacts. Notably, low-ISO and astro-photographs will be smoother and noise will appear less grainy. This method is also orders of magnitude faster than AMaZE and an OpenCL version is provided for GPU execution.
+
+To take advantage of the strengths of different demosaicing methods, "dual demosaicing" modes have been added in a similar manner as Rawtherapee, allowing you to use different algorithms on different parts of the image, depending on content. 
+
+### Details mask refinement
+
+A new parametric masking option has been added to include or exclude sharp edges and details from parametric masks. This is mostly useful to apply sharpening and blurring effects that ignore out-of-focus (bokeh) regions or, alternatively, to sharpen only the blurry parts, preventing over-sharpening of in-focus regions. This option is only available for RAW files as it depends on details from demosaic stage.
+
+## Masks refactoring
+
+The masks GUI has been improved. The "reversed" blending modes are now accessible from an "invert" icon and the blurring (using gaussian blur) and feathering (using edge-aware guided filter) mask refining steps can be applied in a user-defined order.
+
+The darktable default for blending modes is to apply the module output over the module input with some optional transparency (using classical [alpha compositing](https://en.wikipedia.org/wiki/Alpha_compositing) and assuming a fully opaque input). The reversed modes will instead apply the input over the output. 
+
+As of darktable 3.4, the mask feathering (using a guided filter) was always applied before the mask blurring (using a gaussian blur). Both techniques aim at ensuring contiguous image regions are masked continuously, without holes, especially in the presence of noise and when using chromaticity masks. However, the gaussian blur is not edge-aware and can leak out of the boundaries of masked regions, producing edge halos in some situations, whereas the guided filter is taped closely to edges. While the gaussian blur is more effective to deal with noise, using it after the guided filter voids its edge-awareness. Applying in reversed order may help to more aggressively correct noise while still retaining the edge-aware properties of the guided filter (use small blurring radii).
```

Comment:
I'm not a native speaker, but I have problems with this sentence:

> While the gaussian blur is more effective to deal with noise, using it after the guided filter voids it edge-awareness.

- Shouldn't _Gaussian_ be spelt with a capital letter?
- _it_ and _its_ in the sentence: the edge-aware filter is the guided one, so logic says the _its_ refers to the guided filter; however, the _it_ of _using it_ seems to refer back to the Gaussian.